index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Persaonal Portfolio</title>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap");
+    </style>
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body background="./images/git.png"></body>
+    <ul class="flex-container" >
+      
+        <li><h1 class="welcome">Welcome</h1></li>
+
+        <li><i class="fa-solid fa-bars fa-2x"></i></li>
+        
+    </ul>
+    <header class="header-section">
+      <h2 class="header-section-title">
+        <p class="header-name">Hey there. I’m Coral</p>
+        <p class="header-name-header-title">I’m a software developer</p>
+      </h2>
+    </header>
+    <p class="paragraph">
+      I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you
```

Comment:
- [ ] As requested by the first reviewer please use the background image to match with the Figma design. You can download the whole background image and use it.
- [ ] [Optional] Please adjust the gap between the social media icons.
### Your design

![his](https://user-images.githubusercontent.com/81727988/213873673-61c14afa-b420-4dce-9279-218a4d4c8bd8.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213873687-dccff027-6f01-49e3-957f-13f76e60fd76.jpg)